apps/bfd-pipeline/bfd-pipeline-shared-utils/pom.xml

```diff
@@ -86,7 +86,11 @@
 			<artifactId>junit-jupiter-engine</artifactId>
 			<scope>test</scope>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>gov.cms.bfd</groupId>
+            <artifactId>bfd-shared-test-utils</artifactId>
```

Comment:
any reason why the scope on the usages of bfd-shared-test-utils are not test?

Reply:
PipelineTestUtils uses this, and PipelineTestUtils has a comment about being left in src/main to allow it to be used in other modules tests without classpath shenanigans. It may or may not be useful to fix that so this could be better scoped, not sure how much work it'd be, but I'd recommend lumping that (and the below issue that prevents the server-war from being test-scoped) into a tech debt item since they both sound a little hairy and and existed before this change.

I did audit the other instances of bfd-shared-test-utils and found one other spot where I could add a test scope though, so good reminder.

Reply:
Added a comment to the pom about this.

Reply:
Quick look says that PipelineTestUtils probably needs to be put in a new submodule of pipeline that gets imported as a test-only artifact to the other submodules, since things in test directories cant be shared with other test directories in the same parent module, and thats what this was getting around. I'll play around with doing this and see how it looks.

Reply:
Was able to get around this by making a new module in pipeline, moving PipelineTestUtils to it, and then scoping that and bfd-shared-test-utils to test. This takes care of scoping issues in pipeline, at least.